go.mod

```diff
@@ -45,11 +45,11 @@ require (
 	github.com/tensorflow/tensorflow v2.1.0+incompatible
 	github.com/yahoojapan/gongt v0.0.0-20190517050727-966dcc7aa5e8
 	github.com/yahoojapan/ngtd v0.0.0-20190510080733-0c37ddc5e720
-	go.uber.org/automaxprocs v1.2.0
-	golang.org/x/sys v0.0.0-20200121082415-34d275377bf9
+	go.uber.org/automaxprocs v1.3.0
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blueviolet.svg?style=flat-square)
Is it okay to upgrade automaxprocs?